docs/machine-learning/resources/transforms.md

```diff
@@ -20,18 +20,18 @@ Other data transformations don't require training data. For example: the <xref:M
 
 ## Column mapping and grouping
 
-| Transform | Definition |
-| --- | --- |
-| <xref:Microsoft.ML.TransformExtensionsCatalog.Concatenate%2A> | Concatenate one or more input columns into a new output column |
-| <xref:Microsoft.ML.TransformExtensionsCatalog.CopyColumns%2A> | Copy and rename one or more input columns |
-| <xref:Microsoft.ML.TransformExtensionsCatalog.DropColumns%2A> | Drop one or more input columns |
-| <xref:Microsoft.ML.TransformExtensionsCatalog.SelectColumns%2A> | Select one or more columns to keep from the input data |
+| Transform | Definition | ONNX Exportable |
+| --- | --- | --- |
+| <xref:Microsoft.ML.TransformExtensionsCatalog.Concatenate%2A> | Concatenate one or more input columns into a new output column | Yes |
+| <xref:Microsoft.ML.TransformExtensionsCatalog.CopyColumns%2A> | Copy and rename one or more input columns | Yes |
+| <xref:Microsoft.ML.TransformExtensionsCatalog.DropColumns%2A> | Drop one or more input columns | No |
+| <xref:Microsoft.ML.TransformExtensionsCatalog.SelectColumns%2A> | Select one or more columns to keep from the input data | Yes |
 
 ## Normalization and scaling
 
-| Transform | Definition |
-| --- | --- |
-| <xref:Microsoft.ML.NormalizationCatalog.NormalizeMeanVariance%2A> | Subtract the mean (of the training data) and divide by the variance (of the training data) |
+| Transform | Definition | ONNX Exportable |
```

Comment:
Are normalization / scaling exportable to ONNX?

Reply:
They are. Bri asked and I double checked in the code itself. It looks like not all the xml docs in the machine learning repo itself are up to date.